packages/config/src/index.ts

```diff
@@ -25,6 +25,6 @@ export type {
   JsonObject,
   JsonPrimitive,
   JsonValue,
-} from './deprecatedTypes';
```

Comment:
This should remain as this signals that users should update their imports to @backstage/types instead